postman/Login_test

```diff
@@ -0,0 +1,75 @@
+{
+	"info": {
+		"_postman_id": "cdaee4f6-9167-4df8-8e54-3f10e5dcdab5",
+		"name": "Login_Form",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Post",
+			"item": [
+				{
+					"name": "API test",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Valid request\", function () {\r",
+									"    pm.response.to.have.status(200);\r",
+									"});\r",
+									"pm.test(\"email\", function () {\r",
+									"    pm.expect(pm.response.text()).to.include(\"lm.mishra56@gmail.com\");\r",
+									"});\r",
+									"\r",
+									"const jsonData = pm.response.json();\r",
+									"pm.test(\"Test data type of the response\", () => {\r",
+									"  pm.expect(jsonData.email).to.be.a(\"string\");\r",
+									"  pm.expect(jsonData.password).to.be.a(\"string\");\r",
```

Comment:
password is not returned in the response. One of your tests was failing due to this